static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -160,24 +145,6 @@ const createTestAndVariantId = (campaignCodePrefix, campaignID, id) =>
 
 const makeEvent = (id: string, event: string): string => `${id}:${event}`;
 
-const registerIframeListener = (iframeId: string) => {
```

Comment:
iframes are currently unused, so let's clean this up. This stuff can always be added back if we change our minds, but I think they performed poorly?